tests/console/journal_check.pm

```diff
@@ -99,7 +99,7 @@ sub run {
             my $failed_service_output = script_output("systemctl status $service -l || true");
             foreach my $bsc (@matched_bugs) {
                 if ($failed_service_output =~ $bug_pattern->{$bsc}->{description}) {
-                    record_info('Softfail', "Service: $service failed due to $bsc\n$failed_service_output", result => 'softfail');
+                    record_soft_failure("$reference", "$service failed due to $bsc\n$failed_service_output");
```

Comment:
I think this accepts only one argument. Also `$reference` isn't defined here?

Reply:
@Vogtinator yeah, my bad :) I updated the code now and the makefile to reflect the changes

Reply:
Still uses two arguments

Reply:
Should be good now :), I'll follow up with the rest once this is merged